attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -363,7 +373,18 @@ organization_items = [
 ]
 
 // All items
-items = [animal_estimates_disaggregated, organization_items] -> List.flatten -> List.map({|item| [item.name, item]})  -> Dict.fromList
+addId = {|xs|
+  l = List.length(xs)
+  helper_list = List.upTo(0, l-1)
+  result = List.map(helper_list, {|i| 
+    obj = xs[i]
+    obj2 = ({id: "relative_value_animals_2023_04_n" + i})
```

Comment:
You're going to need to still map these to legit names and stuff. My guess is that this method will be a pain for that.

Later on, the IDs will be the key part of the forecast, so important to not have change. (Or, later, if we allow for a way to roll back an estimate)

Reply:
These are already mapped to legit names, though.

Reply:
Do you mean the variable names? Or that they have names with strings? Those wouldn't work as the IDs.

Reply:
We need something to match a page like this (the data still needs to be put together, perhaps by this model)

<img width="1511" alt="image" src="https://user-images.githubusercontent.com/377065/233738259-f1a200bc-d92e-4c71-817a-c22e9b51b39b.png">

Look at how this is done. We need something similar.
https://github.com/quantified-uncertainty/squiggle/blob/develop/packages/relative-values/models/src/macrovariables.ts

Reply:
I saw how this works in /squiggle. If you match them, it does work for now, so you don't need to worry about this now. But later on, we could definitely want a change here, if we use this in larger environments.

Reply:
Ok, invested a few mins into having proper ids.